src/data/reference/hi.json

```diff
@@ -661,7 +716,11 @@
             "returns": "p5 ऑब्जेक्ट"
         },
         "frameRate": {
-            "description": "प्रति सेकंड प्रदर्शित फ़्रेमों की संख्या निर्दिष्ट करें। उदाहरण के लिए, फ्रेमरेट (30) फ़ंक्शन कॉल प्रति सेकंड 30 बार ताज़ा करने का प्रयास करेगी। यदि प्रोसेसर तेजी से निर्दिष्ट दर बनाए रखने के लिए पर्याप्त नहीं है, तो प्रति सेकंड फ्रेम दर हासिल नहीं की जाएगी। सेटअप () के भीतर प्रति सेकंड फ्रेम दर को परिभाषित करने की सिफारिश की जाती है। डिफ़ॉल्ट दर प्रति सेकंड 60 गुना है। यह setFrameRate (वैल) के समान है। बिना तर्क के फ्रेमरेट () फ़ंक्शन को कॉल करने से वर्तमान दर वापस आ जाती है। यह getFrameRate () के समान है। फ़्रेमरेट () फ़ंक्शन को उन तर्कों के साथ कॉल करना जो टाइप नंबर के नहीं हैं या सकारात्मक नहीं हैं, वे वर्तमान दर को भी वापस कर देंगे।",
+            "description": [
+                "प्रत्येक सेकंड में प्रदर्शित होने वाले फ़्रेमों की संख्या निर्दिष्ट करता है। उदाहरण के लिए, फ़ंक्शन कॉल फ़्रेमरेट (30) एक सेकंड में 30 बार ताज़ा करने का प्रयास करेगा। यदि प्रोसेसर निर्दिष्ट दर को बनाए रखने के लिए पर्याप्त तेज़ नहीं है, तो फ़्रेम दर नहीं होगी हासिल किया। फ्रेम दर को <a href=\"#/p5/setup\">setup()</a> के भीतर सेट करने की अनुशंसा की जाती है। डिफ़ॉल्ट फ्रेम दर प्रदर्शन की फ्रेम दर पर आधारित होती है (यहां इसे \" भी कहा जाता है) ताज़ा दर\"), जो अधिकांश कंप्यूटरों पर 60 फ़्रेम प्रति सेकंड पर सेट है। 24 फ़्रेम प्रति सेकंड (फ़िल्मों के लिए सामान्य) या इससे अधिक की फ़्रेम दर सुचारू एनिमेशन के लिए पर्याप्त होगी। यह setFrameRate(val) के समान है। ",
```

Comment:
**Dont covert / Translate Function names, variable name or anything related to code or referrences specific to Processing / Documentation of P5.js**

Reply:
ok